package.json

```diff
@@ -133,15 +132,16 @@
     "setup-polly-jest": "0.11.0",
     "ts-node": "10.9.2",
     "typescript": "5.3.3",
+    "vite": "5.0.8",
     "vite-tsconfig-paths": "4.2.2",
     "vitest": "0.34.2"
   },
   "engines": {
-    "npm": ">=8.0.0 <10.0.0",
-    "node": "^20.0.0 || ^22.0.0",
-    "pnpm": "~8.12.0"
+    "npm": ">=10.8.3",
+    "node": "^22.9.0",
+    "pnpm": "~9.12.0"
   },
-  "packageManager": "pnpm@8.12.0",
+  "packageManager": "pnpm@9.12.0",
```

Comment:
Is updating pnpm (which means updating lockfile) needed for this PR? If it's unrelated, we should probably not mix these two. If we look for any regression, we wouldn't look at additive-only change (adding dockerfile) to be actually impacting app runtime (which does pnpm lock resolution)

Reply:
tested with old versions, appeared to work, force pushed to "squash" all the random commits I had